src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/Marshal/GetExceptionCodeTests.cs

```diff
@@ -36,7 +36,7 @@ public void GetExceptionCode_NormalExceptionInsideCatch_ReturnsExpected(int hres
             Assert.Equal(0, Marshal.GetExceptionCode());
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotNativeAot))]
```

Comment:
Should this be conditioned on COM?

Reply:
I don't think so. There is nothing in `GetExceptionCode()` that needs COM. However, NativeAot's implementation is to throw because it is an obsolete and therefore not implemented method.